bitnami/rabbitmq/README.md

```diff
@@ -91,6 +91,7 @@ The following table lists the configurable parameters of the RabbitMQ chart and
 | `memoryHighWatermark.type`                | Memory high watermark type. Either `absolute` or `relative`                                                          | `relative`                                                   |
 | `memoryHighWatermark.value`               | Memory high watermark value                                                                                          | `0.4`                                                        |
 | `plugins`                                 | List of plugins to enable                                                                                            | `rabbitmq_management rabbitmq_peer_discovery_k8s`            |
+| `communityPlugins`                        | List of custom plugins (URLs) to be downloaded during container initialization                                       | `nil`                                                        |
```

Comment:
I think that having plugins, communityPlugins and extraPlugins deserves having a small documentation section where the differences are better explained.